packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -130,6 +130,15 @@ function SignUpStartInternal(): JSX.Element {
   const isLegalConsentEnabled = userSettings.signUp.legal_consent_enabled;
   const oidcPrompt = ctx.oidcPrompt;
 
+  const onlyLegalAcceptedMissing = useMemo(
+    () =>
+      signUp.missingFields &&
+      signUp.missingFields.length === 1 &&
+      signUp.missingFields[0] === 'legal_accepted' &&
+      signUp.unverifiedFields &&
+      signUp.unverifiedFields.length === 0,
+    [signUp.missingFields, signUp.unverifiedFields],
+  );
```

Comment:
This is already being done on `SignUpContinue`, but wasn't handled on the root sign-up